cmd/controller-manager/app/controller_manager.go

```diff
@@ -300,7 +300,7 @@ func StartControllers(s *options.ControllerManagerServer,
 		}
 
 		glog.V(5).Info("Running controller")
-		go serviceCatalogController.Run(stop)
+		go serviceCatalogController.Run(int(s.ConcurrentSyncs), stop)
```

Comment:
Any idea why? I don't see why this argument is special. It's still based on a flag like everything else.